### PR TITLE
Feature/first confirmed speakers

### PR DESCRIPTION
**Pull request by anyulled**
Add first accepted & confirmed speakers

- Victor Farcic
- Thomas Vitali
- Alex Soto
- Aurea Muñoz

## Speakers
image

## Talk
image

### CODE REVIEW

**Comment by icougil** [comment]
I am the only one who sees big "empty spaces" when having a look in a particular talk? 🤔

image

image

**Comment by anyulled** [comment]
> I am the only one who sees big "empty spaces" when having a look in a particular talk? 🤔
> 
> image
> 
> image

I made some fixes, this is how it displays now

image

image

File: 2021/infoTalk.html [hunk 1]
```diff
@@ -192,35 +195,35 @@ <h3 class="description"></h3>
...
                                 } else {
                                     $('#session-type').append('Workshop');
                                 }
 
-                                var updateSpeakerPage = function () {
-                                    for (var i = 0; i < window.SPEAKERS.length; i++) {
-                                        var speaker = window.SPEAKERS[i],
-                                            $container = $("#speaker-page-"+i),
+                                const updateSpeakerPage = function () {
+                                    $("article").addClass("hidden");
```
Comment (anyulled) [inline 2021/infoTalk.html:205]: hiding all the speakers

File: 2021/infoTalk.html [hunk 2]
```diff
@@ -192,35 +195,35 @@ <h3 class="description"></h3>
...
+                                                    if (social === 'twitter') {
+                                                        link = link.replace('\@', '')
+                                                        if (!link.startsWith('http://') && !link.startsWith('https://')) {
+                                                            link = 'http://twitter.com/' + link
+                                                        }
                                                     }
-                                                  }
-                                                  $container.find("." + social).show();
-                                                  $container.find("." + social + " a").attr("href", link);
+                                                    $container.removeClass("hidden");
```
Comment (anyulled) [inline 2021/infoTalk.html:218]: showing only the ones on the `window.SPEAKERS` variable.

File: 2021/infoTalk.html [hunk 3]
```diff
@@ -192,35 +195,35 @@ <h3 class="description"></h3>
...
+                                                    $container.find("." + social).show();
+                                                    $container.find("." + social + " a").attr("href", link);
                                                 }
 
                                             };
 
                                         $container.find('.speaker-image').attr("src", $('.speaker-image').attr("data-image-base") + speaker.image);
+                                        $container.find('.speaker-image').attr("alt", speaker.name);
```
Comment (anyulled) [inline 2021/infoTalk.html:226]: adding `alt` descriptions to images

File: 2021/infoTalk.html [hunk 4]
```diff
@@ -51,7 +51,7 @@
             <div class="col-lg-9 col-sm-9 col-xs-12 site-content-left">
                 <div class="category-post-list post-list single-list">
 
-                    <article cass="post-294 post type-post status-publish format-standard has-post-thumbnail hentry category-culturel tag-business tag-news tag-congress tag-entertainment">
+                    <article class="post-294 post type-post status-publish format-standard has-post-thumbnail hentry category-culturel tag-business tag-news tag-congress tag-entertainment">
```
Comment (abelsromero) [inline 2021/infoTalk.html:54]: This is it right? All the other changes r just formatting

Comment (anyulled) [reply 2021/infoTalk.html:54]: Yes, the attribute had a typo, the correct one is `class`. You can check the preview in the screenshot I attached above.

**Comment by icougil** [comment]
@anyulled my friend, I think now the Speakers look really great... but now we can't see anything when you click on an specific talk 😅
Example on [Alex's talk](http://127.0.0.1:4000/2021/infoTalk.html?id=60c112eb3be4300877923156)

image

**Review by icougil — CHANGES_REQUESTED** [review]
Description of the talks are not appearing now 😅

**Comment by anyulled** [comment]
> Description of the talks are not appearing now 😅

I'll give it a look today

**Review by icougil — APPROVED** [review]
Now it looks perfect, thx my friend @anyulled ! :-) 
Not sure but it the talk by Viktor is finally should be in english... or in spanish 😅

**Comment by anyulled** [comment]
> Now it looks perfect, thx my friend @anyulled! :-)
> Not sure, but it the talk by Viktor is finally should be in English... or Spanish 😅

That content is not translated yet in the CFP portal. Shall we translate it?

**Pull request merged**